### ISSUE TITLE
Rich text is white font on white background in Windows dark mode

### ISSUE THREAD

**Issue by bmarkides** [AntlerVC/firetable#413]
**Describe the bug**
After setting my Windows system to 'dark mode,' I can no longer see the rich text in the side drawer editor (it appears to be white font on a white background).

**To Reproduce**
1. On a PC, set system settings to 'dark mode'
2. Open Firetable editing drawer to view rich text field
3. You can see the text if you highlight or edit source

**Expected behavior**
White text on dark background in rich text editor (or dark text on white background would be fine).

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome

**Comment by shamsmosowi** [comment]
Hi @bmarkides !

What's the version of Firetable you're currently using?

you can find it by clicking on your user avatar in the top right corner of the home screen, 
![image](https://user-images.githubusercontent.com/13803906/114548577-4d92ca00-9ca3-11eb-8874-977470ee52ed.png)

Note if you don't see a version label, then your Firetable is outdated 😅 and I would recommend updating it

**Comment by bmarkides** [comment]
Hi, @shamsmosowi, thanks for responding! I don't see the version though I did update yesterday - must not have been to the latest version. Will have another go at updating now. Cheers! :)

**Comment by bmarkides** [comment]
It looks like I skipped the final update step of re-building. I just went to try, and I am seeing this error after inputting 'yarn build' in the www directory:

`$ react-scripts build
Failed to compile.

EPERM: operation not permitted, lstat 'D:\path-to-project\firetable\www\build\favicon\firetable.svg'

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

I don't think I ran into this the first time I installed Firetable.
